### ISSUE TITLE
Wrong code path for multinomial logistic regression model

### ISSUE THREAD

**Issue by krislint** [jpmml/jpmml-sparkml#117]
```
            <groupId>org.jpmml</groupId>
            <artifactId>jpmml-sparkml</artifactId>
            <version>1.5.8</version>
```

```
ERROR c.t.t.p.s.i.ModelResultServiceImpl -  convert PMML error
org.apache.spark.SparkException: Multinomial models contain a matrix of coefficients, use coefficientMatrix instead.
	at org.apache.spark.ml.classification.LogisticRegressionModel.coefficients(LogisticRegression.scala:955)
	at org.jpmml.sparkml.model.LogisticRegressionModelConverter.encodeModel(LogisticRegressionModelConverter.java:40)
	at org.jpmml.sparkml.ModelConverter.registerModel(ModelConverter.java:171)
	at org.jpmml.sparkml.PMMLBuilder.build(PMMLBuilder.java:120)
	at org.jpmml.sparkml.PMMLBuilder.buildByteArray(PMMLBuilder.java:256)
	at org.jpmml.sparkml.PMMLBuilder.buildByteArray(PMMLBuilder.java:252)
```

**Comment by vruusmann** [comment]
For starters, please upgrade to the latest JPMML-SparkML version.

> org.apache.spark.SparkException: Multinomial models contain a matrix of coefficients, use coefficientMatrix instead.

This exception is raised here:
https://github.com/jpmml/jpmml-sparkml/blob/1.5.8/src/main/java/org/jpmml/sparkml/model/LogisticRegressionModelConverter.java#L40

TLDR: You are performing multi-class classification against using a binary (ie. two-class) label. That does not make sense.

Please fix your Apache Spark ML application code. Specifically, pay attention to the encoding of the label column, and make sure that the objective function (binary classification vs multi-class classification) agrees with it.
